mozilla-mobile/firefox-android
# Bug 1840094 - [ally] Talkback doesn't recognise the radio button as checked on the "Default search engine" section

Issue link: https://bugzilla.mozilla.org/show_bug.cgi?id=1840094

### What
After this PR talkback will recognise a checked radion button

### Steps to test

1. Turn TalkBack on.
2. Launch Fenix.
3. Go to 3dot menu > Settings > Search > Default search engine.
4. Using the TB gestures, investigate the the search menu. Observe the behaviour.


### Screenshots
| Issue | Fix |
| ----- | --- |
| https://www.loom.com/share/be91ad9ae8224ebbbb749768e5ef3fb6 |  https://www.loom.com/share/b8c5f49563304b118c15705cfcb4b74e?sid=e1959745-656c-4f16-9cc8-b6f2438e8352 |

### Pull Request checklist
 - [x] Changelog: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
 - [x] Accessibility: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features
### After merge
- Milestone: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- Breaking Changes: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### GitHub Automation
